src/Microsoft.Health.Fhir.Core/Microsoft.Health.Fhir.Core.csproj

```diff
@@ -34,15 +34,15 @@
     <PackageReference Include="Ensure.That" Version="9.2.0" />
     <PackageReference Include="FluentValidation" Version="8.6.1" />
     <PackageReference Include="MediatR" Version="9.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.9" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="3.1.9" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.9" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.9" />
-    <PackageReference Include="Microsoft.Health.Abstractions" Version="1.0.80" />
-    <PackageReference Include="Microsoft.Health.Core" Version="1.0.80" />
-    <PackageReference Include="Microsoft.Health.Extensions.DependencyInjection" Version="1.0.80" />
-    <PackageReference Include="Hl7.Fhir.Serialization" Version="1.9.0" />
-    <PackageReference Include="Hl7.FhirPath" Version="1.9.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="$(SdkPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="$(SdkPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="$(SdkPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(SdkPackageVersion)" />
+    <PackageReference Include="Microsoft.Health.Abstractions" Version="$(HealthcareSharedPackageVersion)" />
+    <PackageReference Include="Microsoft.Health.Core" Version="$(HealthcareSharedPackageVersion)" />
+    <PackageReference Include="Microsoft.Health.Extensions.DependencyInjection" Version="$(HealthcareSharedPackageVersion)" />
+    <PackageReference Include="Hl7.Fhir.Serialization" Version="$(Hl7FhirVersion)" />
```

Comment:
Is the change to using a property related to dependabot? Should use use properties across the board for packages that appear in many places?

Reply:
I found in the [shared components repo](https://github.com/microsoft/healthcare-shared-components/pull/135) dependabot was able to detect the variable and update it. It also provided batched the update info for the related packages.

I thought I'd hit the most obvious packages that are updated together which I think will be the bulk. But yes, since its supported we could globally define the package versions, this won't affect the number of PRs generated though